savaslabs/durham-civil-rights-map
# Issue #4649:  Oct 2017 Updates

Redmine [#4649](https://pm.savaslabs.com/issues/4649)

Hi @chrisarusso - 

This PR is the branch for the `oct-update` MultiDev environment and does the following
1. Applies Drupal Core updates (many from `8.1.7` - `8.4.0`)
2. Applies contrib module "Google Analytics" security update ([release notes](https://www.drupal.org/node/2782023))
3. Exports and commits the active configuration

To Test
1. Please go to https://dashboard.pantheon.io/sites/3c90bad2-fdfb-4028-872b-ded17aa47df3#oct-update and confirm you can see the 3 commits applying the above changes
2. Click `site admin` and go to http://oct-update-durham-civil-rights-map.pantheonsite.io/admin/reports/updates and verify that there are no outdated "red" modules
3. Verify the Database is up to date and that the site works as expected

If all looks good then let me know and I will
1. Merge this PR branch into `master`
2. Push the github `master` branch to Pantheon DEV `master` branch
3. Then deploy on Patheon by pushing from DEV -> TEST -> LIVE
